src/safeposix/syscalls/sys_calls.rs

```diff
@@ -41,7 +41,7 @@ use crate::safeposix::filesystem::{decref_dir, metawalk, Inode, FS_METADATA};
 use crate::safeposix::net::NET_METADATA;
 use crate::safeposix::shm::SHM_METADATA;
 
-use std::sync::Arc as RustRfc;
```

Comment:
Was this an extra import we are removing?

Reply:
Yes this was an extra import